### PR TITLE
Check for custom TLD before check against the list of existing TLDS

### PR DESCRIPTION
**Pull request by akost**
That will allow to add custom TLD like `example.com`.

Fixes issue #8.

### CODE REVIEW

File: lib/parseDomain.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ function parseDomain(url, options) {
     tld = domain.match(knownTlds);
```
Comment (jhnns) [inline lib/parseDomain.js]: This can be removed now

**Comment by jhnns** [comment]
Thx for the PR. Just remove the line and I will merge it. :)

**Comment by akost** [comment]
I updated my PR, removed the line.

Thank you!

File: lib/parseDomain.js (hunk 1 above)
Comment (akost) [inline lib/parseDomain.js]: I removed this, could you merge now?

**Pull request merged**